client/app/components/faq/FaqList.js

```diff
@@ -132,8 +132,8 @@ const mapDispatchToProps = dispatch => {
 };
 
 FaqList.propTypes = {
-	// eslint-disable-next-line react/forbid-prop-types
-	currentQuestions: PropTypes.array
+	currentQuestions: PropTypes.array,
+	onFAQLoaded: PropTypes.func.isRequired
```

Comment:
Put the required props above those that aren't so that we can easily see which ones are mandatory props.